BParkingNano/test/run_nano_cfg.py

```diff
@@ -124,6 +124,109 @@
 # Other statements
 from Configuration.AlCa.GlobalTag import GlobalTag
 process.GlobalTag = GlobalTag(process.GlobalTag, globaltag, '')
+# this is for the LowPt energy regression
+process.GlobalTag.toGet = cms.VPSet(
```

Comment:
it seems like the same three lines are added many times? is this correct?

Reply:
@cavalari

Reply:
actually ignore this comment... they are not the same but the naming is extremely close... Can be moved to another file so the cfg remains clean(er)?

Reply:
I propose to leave it there, and once we put the objects in the DB and make a GT out of it, there is no need for these lines.